params/util.go

```diff
@@ -38,6 +39,9 @@ var (
 	TestNetSpuriousDragon = big.NewInt(10)
 	MainNetSpuriousDragon = big.NewInt(2675000)
 
-	TestNetChainID = big.NewInt(3) // Testnet default chain ID
-	MainNetChainID = big.NewInt(1) // Mainnet default chain ID
+	TestNetMetropolisBlock = big.NewInt(math.MaxInt64)
+	MainNetMetropolisBlock = big.NewInt(math.MaxInt64)
+
+	TestNetChainID = big.NewInt(3) // Test net default chain ID
+	MainNetChainID = big.NewInt(1) // main net default chain ID
```

Comment:
I'd recommend dropping all these fields (,aybe the hash needs to stay and using the already existing ChainConfig constants instead.